src/lib/guides/StarterKits/getting-started.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Getting Started with Wharfkit Starter Kits
+category: StarterKits
+published: true
+requiresReview: true
+---
+
+# Introduction to Wharfkit Starter Kits
+
+The Starter kits are a great way to start developing applications on Wharfkit. They offer a convenient way to install all the essential Wharfkit packages at once, while also providing `sessionKit` instances that can be used quickly instantiate sessions in your applications.
+
+The available kits are `@wharfkit/starter`, `@wharfkit/starter-eos`, `@wharfkit/starter-telos`, `@wharfkit/starter-wax`, and `@wharfkit/starter-jungle`.
+
+> **NOTE**: This guide is current as of December 2023 and reflects the latest versions of Wharfkit Starter Kits.
+
+## Getting Started
+
+Ensure [Node.js](https://nodejs.org/en/) is installed, as it is a prerequisite for using Wharfkit packages.
```

Comment:
NPM or Yarn is required, but not nodejs specifically? 🤔 

I'm not sure we even need this section TBH. People could be using bun too

Reply:
Not sure that we need it either, but can you even download npm or yarn without Node.js? 😄